st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -82,6 +82,9 @@ def post(self, payload):
             'source_channel': payload.source_channel
         }
 
+        if hasattr(payload, 'notification_route'):
+            context['route'] = getattr(payload, 'notification_route')
```

Comment:
Would probably be better to just always include this attribute and just default it to `None` if no route is specified (`context['route'] = getattr(payload, 'notification_route', None)`).

This way it's more explicit and easier to do Jinja comparisons since the attribute is always present.